### PR TITLE
RHPAM-2532: changing failing test to use ssh instead git protocol

### PR DESCRIPTION
**Pull request by porcelli**

### CODE REVIEW

File: business-central-tests/pom.xml [hunk 1]
```diff
@@ -290,7 +292,7 @@
                 <configuration>
                   <properties>
                     <cargo.jboss.configuration>standalone</cargo.jboss.configuration>
-                    <cargo.jvmargs>-Xmx1024m</cargo.jvmargs>
+                    <cargo.jvmargs>-Xmx1024m -Dorg.uberfire.nio.git.daemon.enabled=true</cargo.jvmargs>
```
Comment (porcelli) [inline business-central-tests/pom.xml]: for an unknown reason, it only works if I set it here, the above systemProperties doesn't work

File: business-central-tests/pom.xml [hunk 2]
```diff
@@ -158,9 +158,10 @@
               <kie.wb.url>http://${container.host}:${container.port}/${deployable.context}</kie.wb.url>
               <kie.controller.url>ws://${container.host}:${container.port}/${deployable.context}/websocket/controller</kie.controller.url>
               <org.uberfire.nio.git.daemon.port>${cargo.git.port}</org.uberfire.nio.git.daemon.port>
+              <org.uberfire.nio.git.daemon.enabled>true</org.uberfire.nio.git.daemon.enabled>
             </systemPropertyVariables>
             <failIfNoTests>false</failIfNoTests>
-            <argLine>-Xmx1024m -Dfile.encoding=UTF-8</argLine>
+            <argLine>-Xmx1024m -Dfile.encoding=UTF-8 -Dorg.uberfire.nio.git.daemon.enabled=true</argLine>
```
Comment (porcelli) [inline business-central-tests/pom.xml]: for an unknown reason, it only works if I set it here, the above systemProperties doesn't work

**Comment by paulovmr** [comment]
Unrelated test failure.

**Comment by tomasdavidorg** [comment]
@porcelli Why didn't you use the http protocol? The fix is not ideal since the git protocol is disabled by default.

**Pull request merged**